### PR TITLE
Servlet for admins to upload files

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #112 

Changes: Added servlet to upload file.
@Orbiter Please can you look and suggest if I am missing something. 
Thanks!

### CODE REVIEW

**Comment by DravitLochan** [comment]
@dynamitechetan @saurabhjn76 @jyoth1raditya @betterclever Please review!

File: src/ai/susi/server/api/aaa/UploadSettingsService.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by dravit on 28/6/17.
+ * http://127.0.0.1:4000/aaa/uploadSettings.json
```
Comment (dynamitechetan) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: Please also add which parameters are to be sent with requests.

File: src/ai/susi/server/api/aaa/UploadSettingsService.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        String fileName = null;
+        //Get all the parts from request and write it to the file on server
+        try {
+            HttpServletRequest request = post.getRequest();
+            for (Part part : request.getParts()) {
+                fileName = getFileName(part);
+                part.write( path + File.separator + fileName);
+                request.setAttribute("message", fileName + " File uploaded successfully!");
+            }
+            result.put("success","true");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: boolean! not String!

File: src/ai/susi/server/api/aaa/UploadSettingsService.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        } catch (IOException e) {
+            e.printStackTrace();
+            result.put("success","false");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: boolean

File: src/ai/susi/server/api/aaa/UploadSettingsService.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        } catch (ServletException e) {
+            e.printStackTrace();
+            result.put("success","false");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: boolean

File: src/ai/susi/server/api/aaa/UploadSettingsService.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
+/**
```
Comment (aayusharora) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: Can you please remove these above comments, please maintain the code quality.

**Comment by aayusharora** [comment]
@DravitLochan please squash your commits!!
Thanks.

File: src/ai/susi/server/api/aaa/UploadSettingsService.java (hunk 5 above)
Comment (DravitLochan) [inline src/ai/susi/server/api/aaa/UploadSettingsService.java]: It's a copyright header which all the servlets from aaa must have. I do not see where it deals with  code quality?

**Pull request merged**